js/ui/appDisplay.js

```diff
@@ -624,6 +624,7 @@ class AppDisplay extends BaseAppView {
                         return;
                     }
                     icon = new FolderIcon(item, this);
+                    icon.connect('clicked', () => { this._bgAction.release(); })
```

Comment:
Style issue, the correct style should be:

```javascript
icon.connect('clicked', () => this._bgAction.release());
```

Reply:
Updated.